src/tester.ts

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+import 'colors';
+import { findFiles } from './main';
+import yargs = require('yargs');
```

Comment:
This is not correct. It should be like

```ts
import yargs from 'yargs';
```

or 

```ts
import * as yargs from 'yargs';
```

Reply:
Done, thanks. The second is the syntax I should have used.